heap/max_heap.py

```diff
@@ -50,21 +50,30 @@ def _bubble_up(self, index):
         # print('array', self.storage)
 
     def _sift_down(self, index):
+        # index is parent
         parent = index
+        # get the left and right child
         left_child = 2 * parent + 1
         right_child = 2 * parent + 2
+        # check if left and right child are in the array
         if left_child > len(self.storage) - 1:
```

Comment:
nice! remember that `self.storage[left_child]` will throw an error if there is no child, which would break the program. well done.